doc/rst/platforms/infiniband.rst

```diff
@@ -25,30 +25,49 @@ Chapel with InfiniBand support, set:
       export CHPL_COMM=gasnet
       export CHPL_COMM_SUBSTRATE=ibv
 
-Alternatively, when running on a Cray CS or HPE Apollo system
-``CHPL_HOST_PLATFORM`` can instead be set, in which case the comm
+Alternatively, when running on an HPE Apollo or Cray CS system
+``CHPL_HOST_PLATFORM`` can be set instead, in which case the comm
 and substrate settings will be inferred.
 
+For HPE Apollo:
+
+.. code-block:: bash
+
+      export CHPL_HOST_PLATFORM=hpe-apollo
+
 For Cray CS:
 
 .. code-block:: bash
 
       export CHPL_HOST_PLATFORM=cray-cs
 
-For HPE Apollo:
+
+UCX alternative
+---------------
+
+In rare cases, we have had users who are unable to use the ``ibv``
+conduit on their InfiniBand systems.  In such cases, the ``ucx``
+conduit is an alternative, which requires setting:
 
 .. code-block:: bash
 
-      export CHPL_HOST_PLATFORM=hpe-apollo
+      export CHPL_COMM=gasnet
+      export CHPL_COMM_SUBSTRATE=ibv
+
+(and, most likely, ``export CHPL_LAUNCHER=gasnetrun_ucx`` when you
+reach the following section).
 
+Note that the GASNet team currently considers the ``ucx`` conduit
+"experimental" in the sense that it is feature complete but has not
+received as much performance-oriented work as ``ibv`` has.
```

Comment:
I'd personally be hesitant to add ucx without more disclaimers. So far as I know there's no nightly chapel testing with it. And last time I ran correctness testing there were some failures. From https://github.com/chapel-lang/chapel/pull/17152 maybe it was just an AM overload test that hits the lack of flow control for AMs, but I'm not sure.

(I'm not sure how hard it would be to add ucx testing. It was pretty easy to run in 17152, but that was a Cray CS with slurm. Not sure how difficult it'd be to set up on Apollo w/ slurm.)